README.md

```diff
@@ -1 +1,176 @@
-# angular-link-dom-manipulation-readme
\ No newline at end of file
+# Using the link function for DOM manipulation
+
+## Overview
+
+We've had a look at the link function, now lets do some DOM manipulation with it.
+
+## Objectives
+
+- Use a native event API
+- Use a Controller inside Directive
+- Update Controller layer using native events
+
+## Native Events
+
+Now that we've got access to the actual DOM nodes, we can do our own, manual DOM events on the elements. Whilst generally we'd use `ng-click` and let Angular do the dirty work for us, we can do it ourselves manually if we are using 3rd-party plugins that aren't compatible with Angular.
+
+Let's take a simple directive:
+
+```js
+function SomeDirective() {
+	return {
+		template: [
+			'<div>',
+				'<span>Click on me!</span>',
+			'</div>'
+		].join(''),
+		link: function (scope, elem, attrs) {
+
+		}
+	}
+}
+
+angular
+	.module('app')
+	.directive('someDirective', SomeDirective);
+```
+
+Now it's important to note that `elem` isn't *actually* the raw DOM node. It is a jqLite (or jQuery if you have loaded jQuery before Angular) element. However, we can access the raw DOM node via `elem[0]`.
```

Comment:
This may be confusing if you're not familiar with jqLite - might need to add some context around this in an aside section.